### ISSUE TITLE
Suggestion: Savable Giving list of items to inventory

### ISSUE THREAD

**Issue by Bazimek** [alexander171294/RustMon#32]
Savable list of items for giving it to a player with one click.

Then just like right click on a nickname choose give items -> saved pack of items.

**Comment by alexander171294** [comment]
Yes I was thinking of adding a tray of buttons for give objects, your idea to add packs are really good.

Thanks for your suggestion.
